index.html

```diff
@@ -0,0 +1,195 @@
+                </div>
+              </div>
+            </div>
+            <div class="panel-wrapper">
+              <a href="#show" class="show" id="show">Show more</a>
+              <a href="#hide" class="hide" id="hide">Show less</a>
+              <div class="panel">
+                <p id="visits">68,481 vists 8 years ago</p>
+                <div id="descCont"><p id="description">How do you contribute to open source?  In this video, I'll describe the easiest way to make a simple contribution to an open source project -- without using the command line, a text editor, "forking a repo", or even touching Git outside of Github.  I'll also describe what kinds of contributions open source software projects are typically looking for.  As an example, I use submitting a student solution to a web development exercise from The Odin Project, a free online curriculum for learning web development with Ruby on Rails.  You can find more information about that project at <a href="http://theodinproject.com">http://theodinproject.com</a></p></div> 
+            </div>
+            <div class="fade"></div>
+          </div>
+        </div>
+      </main>
+      <aside>
+        <div class="asideCont">
+          <div class="thumbCont">
+            <img class="thumbImg" src="/assets/aside-img_1.png" alt="">
+            <div class="thumbDescCont">
+              <p class="thumbTitle">Contributing to Open Source Part II: The Real Way</p><p class="thumbCreator">The Odin Project</p>
+              <div class="smllThmbTxt">
+                <p class="visits">38,831 visits</p><div class="dash"></div><p class="time">8 years ago</p>
+              </div>
+          </div>
+          </div>
+          <div class="thumbCont">
+            <img class="thumbImg" src="/assets/aside-img_2.png" alt="">
+            <div class="thumbDescCont">
+              <p class="thumbTitle">Contributing to Open Source on GitHub For Beginners</p><p class="thumbCreator">Kent C. Dodds</p>
+              <div class="smllThmbTxt">
+                <p class="visits">90,232 visits</p><div class="dash"></div><p class="time">3 years ago</p>
+              </div>
+          </div>
+          </div>
+          <div class="thumbCont">
+            <img class="thumbImg" src="/assets/aside-img_3.png" alt="">
+            <div class="thumbDescCont">
+              <p class="thumbTitle">GitHub and Open Source Documentation</p><p class="thumbCreator">Rahul Karn</p>
+              <div class="smllThmbTxt">
+                <p class="visits">2,125 visits</p><div class="dash"></div><p class="time">4 years ago</p>
+              </div>
+          </div>
+          </div>
+          <div class="thumbCont">
+            <img class="thumbImg" src="/assets/aside-img_4.png" alt="">
+            <div class="thumbDescCont">
+              <p class="thumbTitle">GitHub How to build a strong community for your open...</p><p class="thumbCreator">Wix Engineering Tech Talks</p>
```

Comment:
I suggest starting a new paragraph in a new line for better code readability

Reply:
This is a better review comment because it is clear and there is "why" mentioned in the comment, an opportunity for the student to learn why the change is requested.